File: Programmers/Lv.0/그림 확대/181836.java
class Solution {
    public String[] solution(String[] picture, int k) {
        String[] answer = new String[picture.length * k];
        
        for (int a=0; a<picture.length; a++) {
            String[] str = picture[a].split("");
            String s = "";
            
            for (int b=0; b<str.length; b++) {
                s += str[b].repeat(k);
            }
            
            for (int c=0; c<k; c++) {
                answer[(a * k) + c] = s;
            }
        }
        
        return answer;
    }
}

File: Programmers/Lv.0/그림 확대/181836.js
function solution(picture, k) {
    const answer = [];
    
    for (let a=0; a<picture.length; a++) {
        const str = picture[a].split('');
        let s = '';
        
        str.forEach((v, i) => s += v.repeat(k));
        
        for (let b=0; b<k; b++) answer[(a * k) + b] = s;
    }
    
    return answer;
}
